routes/console.php

```diff
@@ -37,7 +37,7 @@
 
         $wasPlayed = !is_null($mission->last_played);
         if (!$wasPlayed || ($wasPlayed && ($mission->last_played < $header->starts_at))) {
-            $prev = $mission->last_played->toDateTimeString();
+            $prev = $wasPlayed ? $mission->last_played->toDateTimeString() : "null";
```

Comment:
"null" 🤔

Reply:
php won't automatically convert null to a string (just leaves a blank space)